akka-docs/src/main/paradox/scala/actors.md

```diff
@@ -480,50 +617,58 @@ actors which are traversed in the sense that if a concrete name lookup fails
 negative result is generated. Please note that this does not mean that delivery
 of that reply is guaranteed, it still is a normal message.
 
-@@snip [ActorDocSpec.scala]($code$/scala/docs/actor/ActorDocSpec.scala) { #identify }
+Scala
+:  @@snip [ActorDocSpec.scala]($code$/scala/docs/actor/ActorDocSpec.scala) { #identify }
+
+Java
+:  @@snip [ActorDocTest.java]($code$/java/jdocs/actor/ActorDocTest.java) { #import-identify #identify }
 
 You can also acquire an `ActorRef` for an `ActorSelection` with
 the `resolveOne` method of the `ActorSelection`. It returns a `Future`
-of the matching `ActorRef` if such an actor exists. It is completed with
+of the matching `ActorRef` if such an actor exists. @java[(see also 
+@ref:[Java 8 and Scala Compatibility](scala-compat.md) for Java compatibility).] It is completed with
 failure [[akka.actor.ActorNotFound]] if no such actor exists or the identification
 didn't complete within the supplied *timeout*.
 
 Remote actor addresses may also be looked up, if @ref:[remoting](remoting.md) is enabled:
 
-@@snip [ActorDocSpec.scala]($code$/scala/docs/actor/ActorDocSpec.scala) { #selection-remote }
+Scala
+:  @@snip [ActorDocSpec.scala]($code$/scala/docs/actor/ActorDocSpec.scala) { #selection-remote }
+
+Java
+:  @@snip [ActorDocTest.java]($code$/java/jdocs/actor/ActorDocTest.java) { #selection-remote }
 
 An example demonstrating actor look-up is given in @ref:[Remoting Sample](remoting.md#remote-sample).
 
 ## Messages and immutability
 
 @@@ warning { title=IMPORTANT }
 
-Messages can be any kind of object but have to be immutable. Scala can’t enforce
-immutability (yet) so this has to be by convention. Primitives like String, Int,
+Messages can be any kind of object but have to be immutable. @scala[Scala] @java[Akka] can’t enforce
+immutability (yet) so this has to be by convention. @scala[Primitives like String, Int,
 Boolean are always immutable. Apart from these the recommended approach is to
 use Scala case classes which are immutable (if you don’t explicitly expose the
-state) and works great with pattern matching at the receiver side.
+state) and works great with pattern matching at the receiver side.]
 
 @@@
 
-Here is an example:
+Here is an example @java[of an immutable message]:
```

Comment:
Yeah it did, so the new commit fixed it now.

![image](https://cloud.githubusercontent.com/assets/7414320/26733268/39d1a84e-47f5-11e7-9c9c-7625b2a14874.png)

![image](https://cloud.githubusercontent.com/assets/7414320/26733127/d0a033ae-47f4-11e7-85ac-f06eb35a9ab1.png)